assets/js/slackbot.js

```diff
@@ -0,0 +1,274 @@
+const { App } = require("@slack/bolt");
```

Comment:
Since we're adding .js files, we should probably add a ticket later on for implementing `eslint` or `prettier`, standard libraries like we have for Helios